src/plugins/intel_cpu/src/transformations/snippets/tpp/op/reduce.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "reduce.hpp"
+
+namespace ov {
+namespace intel_cpu {
+namespace tpp {
+namespace op {
+
+ReduceMax::ReduceMax(const Output<Node>& arg, size_t axis)
+    : UnaryEltwiseTPP(LIBXSMM_MELTW_TYPE_UNARY_REDUCE_X_OP_MAX), ov::snippets::op::ReduceMax(arg, axis) {
+}
+
+std::shared_ptr<Node> ReduceMax::clone_with_new_inputs(const OutputVector& new_args) const {
+    check_new_args_count(this, new_args);
+    const auto& new_op = std::make_shared<ReduceMax>(new_args.at(0), m_axis);
+    new_op->clone_memory_acess_ports(*this);
+    return new_op;
+}
+
+bool ReduceMax::visit_attributes(AttributeVisitor& visitor) {
+    ReduceBase::visit_attributes(visitor);
```

Comment:
I don't get it, could you elaborate please?
If we replace L24 with `ov::snippets::op::ReduceMax::visit_attributes(visitor)` it would be an infinite loop, wouldn't it?